### PR TITLE
Updates file.absent documentation to include details on how to clear contents of directory

### LINKED ISSUE #61291: [DOCS] Improve file.absent description with reference to file.directory
**Description**
I needed to delete the complete contents of the directory, but leave the directory in existence. I hoped that `file.absent` would have a flag for that, but no luck. And then I didn't know where else to look.

**Suggested Fix**
Please add to `file.absent` documentation something like:

In case you want to delete the contents of the directory, but leave the directory in existence use `file.directory` with `clean=True`.

**Type of documentation**
Salt modules

**Location or format of documentation**
Insert page URL if applicable.

**Additional context**

### PR DESCRIPTION
**Pull request by priyank-R**
### What does this PR do?
Updates file.absent documentation to include details on how to clear contents of directory
### What issues does this PR fix or reference?
Fixes: #61291 

### Merge requirements satisfied?
- [ ] Docs

### Commits signed with GPG?
No

Please review [Salt's Contributing Guide](https://docs.saltproject.io/en/master/topics/development/contributing.html) for best practices.

See GitHub's [page on GPG signing](https://help.github.com/articles/signing-commits-using-gpg/) for more information about signing commits with GPG.

### CODE REVIEW

**Comment by priyank-R** [comment]
@barbaricyawps Please check it out. Thanks

**Review by barbaricyawps — APPROVED** [review]
@priyank-R , this looks great to me. I'll see if I can get the maintainers to merge it in.

File: salt/states/file.py [hunk 1]
```diff
@@ -1882,6 +1882,9 @@ def absent(name, **kwargs):
     be deleted. This will work to reverse any of the functions in the file
     state module. If a directory is supplied, it will be recursively deleted.
 
+    If only the contents of the directory need to be deleted but not the directory itself, use :mod:`file.directory
```
Comment (twangboy) [inline salt/states/file.py:1885]: Can we honor the 80 character line limit?

**Review by twangboy — CHANGES_REQUESTED** [review]

File: salt/states/file.py (hunk 1 above)
Comment (Ch3LL) [reply salt/states/file.py:1885]: I added a PR here https://github.com/saltstack/salt/pull/61495

Comment (priyank-R) [reply salt/states/file.py:1885]: Thank you for correcting @twangboy, I'll make sure to follow it now it.

**Pull request merged**